build-release.sh

```diff
@@ -219,13 +219,13 @@ if [ "${build_classical}" == "1" ]; then
   binname="${godot_basename}_linux.x86_64"
   cp out/linux/x86_64/tools/godot.linuxbsd.editor.x86_64 ${binname}
   strip ${binname}
-  zip -q -9 "${reldir}/${binname}.zip" ${binname}
+  7z a -bso0 -bd -mx9 "${reldir}/${binname}.zip" ${binname}
```

Comment:
Can you make this a define like:
```
export ZIP="7z a -bso0 -bd -mx9"
```

to use with `${ZIP}` and `${ZIP} -r`, so we don't have to hardcode the options everywhere?